code/obj/machinery/manufacturer.dm

```diff
@@ -15,12 +15,14 @@
 	var/mode = "ready"
 	var/error = null
 	var/speed = 3
+	var/head_access = list(access_heads)
```

Comment:
useless var, never gets modified?

Reply:
Future-proofing, in case a fabricator needs different access requirements for some reason. EG a more department-specialized fabricator could restrict head access to just the department head and the Cap, and vice versa, a more generalized fabricator could loosen it to include some non-Head jobs.

Reply:
Consider a different name in that case. `unlock_access`?